src/WordTokenizers.jl

```diff
@@ -4,7 +4,12 @@ module WordTokenizers
 using HTML_Entities
 using StrTables
 using Unicode
+using GoogleDrive
```

Comment:
CI is failing. I think this may be the cause.
Can you look into it?

Reply:
yeah my bad